src/main/java/net/sigmarik/abilitymod/AbilityMod.java

```diff
@@ -33,10 +40,99 @@ public class AbilityMod implements DedicatedServerModInitializer {
 
 	public static final double BOAT_ATTRACTION_FACTOR = 0.1;
 
+	public static final int FIRE_RESISTANCE_STATUS_EFFECT = 12;
+
+	public static final int FIRING_CHANCE_PER_TICK = 55;
```

Comment:
No linkage to the trait this parameter corresponds to. Change the name or move it somewhere to make it obvious that it affects "DAMAGED_BY_DAYLIGHT" trait effect.

And also... Why is catching flame on daylight has a probability to happen? It makes sense for zombies, as they need to vary their flames a little so they wouldn't  take damage at the exact same ticks. But I highly doubt there will every be a crowd of players with this trait purposefully syncing their flaming ticks.